### ISSUE TITLE
Last Version Plugin Absolutely Broken

### ISSUE THREAD

**Issue by eflernicolas** [AoiKagase/Amxx-ReviveKit#8]
Now you cannot revive human players, you get this message every time in print chat:

[Revive Kit]: Reviving Player
[Revive Kit]: Failed target reviving.

And the corpse disappear if you are not fast enough to revive, also after of failing revive it disappears.

Edit: the weirdest bug happens:

i changed the cvar rkit_delay_die to 9999 since 0 was causing the above error, my idea is to never have time remaining to revive someone, i dont really like that mechanic and also i think is a lot of load to the sv to have a timer for every player /bot out there.

Now, after setting cvar_rkit_delay_die to 9999,  and restarting a few times the map so cvars finally apply corpses stay on ground, revive bar fills, gives error, millions of corpses overlap server drops to 10 fps, lags like hell.

About the cvar issue: i opened another issue for that since i had problems with cvars in the last version but i couldnt even have the time to write to you about it 

**Kicked: reliable channel overflowed.**

TLDR: 
Cvars dont work
Server drops to 10 fps , inestability and crash/kick "reliable channel overflowed"
Plugin doesnt work anymore

**Comment by AoiKagase** [comment]
> [Revive Kit]: Failed target reviving.

This happens when you are knocked back out of the revive range, or when the target's reviveable time has expired.
The corpse will disappear when the revivable time expires.

Fix the function so that rkit_delay_die = 0 will stop this function.
(We are still investigating the bug in Cvar.)

> Kicked: reliable channel overflowed.

Does it happen if I simply change the cvar?

**Comment by eflernicolas** [comment]
> > [Revive Kit]: Failed target reviving.
> 
> This happens when you are knocked back out of the revive range, or when the target's reviveable time has expired. The corpse will disappear when the revivable time expires.
> 
> Fix the function so that rkit_delay_die = 0 will stop this function. (We are still investigating the bug in Cvar.)
> 
> > Kicked: reliable channel overflowed.
> 
> Does it happen if I simply change the cvar?

at first  rkit_delay_die was set to 0. and the issue still occurred, meaning that corpses would stay on ground, but i would not be able to revive and server would start to lag and fps to drop. its like million corpses are spawning at the area.

**Comment by AoiKagase** [comment]
Check the fix in #9.
With rkit_delay_die = 0, the revive time function should no longer work.
This will also prevent the set_task from being executed, which I believe will mitigate the FPS drop.

**Comment by eflernicolas** [comment]
> Check the fix in #9. With rkit_delay_die = 0, the revive time function should no longer work. This will also prevent the set_task from being executed, which I believe will mitigate the FPS drop.

This worked for me, plugin seems stable now, and i vote for that mechanic to be removed!

Closing this issue and will open another regarding other matters
